bb/src/main/java/com/richardmcdougall/bb/BurnerBoardUtil.java

```diff
@@ -87,4 +95,58 @@ public static final boolean isBBPanel() {
             || BurnerBoardUtil.kIsRPI
         ) ? true : false;
     }
-}
\ No newline at end of file
+
+
+    public static final String publicNameFile = "publicName.txt";
```

Comment:
We could just use a preference setting to store this. no file name required.
I think it's something like this:

SharedPreferences prefs = mMain.getSharedPreferences("names", mMain.MODE_PRIVATE);
String publicName = prefs.getString("publicname", "unknown");